tests/add-user-ui.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { test, expect } from "@playwright/test";
+
+test("has title", async ({ page }) => {
+  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
+
+  // Expect a title "to contain" a substring.
+  await expect(page).toHaveTitle("TS Trainee course");
+});
+
+test("Create button UI", async ({ page }) => {
+  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
+  const el = page.getByRole("button", { name: "Create" });
+  await expect(el).toBeVisible;
+  await expect(el).toHaveClass("btn btn-primary");
+  await expect(el).toHaveCSS("background-color", "rgb(13, 110, 253)");
+  // checking btn reaction to hover
+  await el.hover();
+  await expect(el).toHaveCSS("background-color", "rgb(11, 94, 215)");
+});
+
+test("Cancel button UI", async ({ page }) => {
+  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
+  const el = page.getByRole("link", { name: "Cancel" });
+
+  await expect(el).toHaveClass("btn btn-secondary");
+  await expect(el).toHaveCSS("background-color", "rgb(108, 117, 125)");
+  // checking btn reaction to hover
+  await el.hover();
+  await expect(el).toHaveCSS("background-color", "rgb(92, 99, 106)");
+});
+
+test("User name UI", async ({ page }) => {
+  await page.goto("https://traineeautomation.azurewebsites.net/Forms/AddUser");
+  const el = page.getByLabel("User Name");
```

Comment:
for this task it's okay, but advice for a future, won't relate to labels. Today it's User Name, tomorrow it's Name. And also it could be translated in different languages and it can be "Імʼя користувача" and you will need to check all possible languages. 

Add here a comment, that it should be rewritten using XPath